tests/src/test/spark350/scala/com/nvidia/spark/rapids/ToPrettyStringSuite.scala

```diff
@@ -0,0 +1,140 @@
+      .eval(null).asInstanceOf[UTF8String].toString()
+  }
+
+  test("test show() Array") {
+    val dataType = ArrayType(DataTypes.IntegerType)
+    val v: Array[_] = Array(1, 2, 3, null)
+    // Execute on CPU
+    val cpuResult = executeOnCpuAndReturn(dataType, ArrayData.toArrayData(v))
+
+    val child = GpuBoundReference(0, dataType, true)(NamedExpression.newExprId, "array_arg")
+    val gpuToPrettyStr = GpuToPrettyString(child, Some("UTC"))
+    withResource(
+      GpuColumnVector.from(ColumnVector.fromStrings(cpuResult),
+      DataTypes.StringType)) { expected0 =>
+      val dt = new HostColumnVector.ListType(true,
+        new HostColumnVector.BasicType(true, DType.INT32))
+      withResource(GpuColumnVector.from(ColumnVector.fromLists(dt,
+        v.toList.asJava),
+        ArrayType(DataTypes.IntegerType))) { input =>
+        val batch = new ColumnarBatch(List(input).toArray, 2)
+        checkEvaluation(gpuToPrettyStr, expected0, batch)
+      }
+    }
+  }
+
+  test("test show() Map") {
+    val dataType = MapType(DataTypes.IntegerType, DataTypes.IntegerType)
+    val map = Map(1 -> 2, 2 -> 3, 4 -> null)
+    val cpuResult = executeOnCpuAndReturn(dataType, ArrayBasedMapData(map))
+
+    val child = GpuBoundReference(0, dataType, true)(NamedExpression.newExprId, "map_arg")
+    val gpuToPrettyStr = GpuToPrettyString(child, Some("UTC"))
+
+    withResource(
+      GpuColumnVector.from(
+        ColumnVector.fromStrings(cpuResult), DataTypes.StringType)) {
+      expected0 =>
+        val list1 = map.map {
+          case (k, v) =>
+            new HostColumnVector.StructData(Array[Integer](k, v.asInstanceOf[Integer]): _*)
+        }.toList.asJava
+        val structType = new HostColumnVector.StructType(true,
+          List[HostColumnVector.DataType](new HostColumnVector.BasicType(true, DType.INT32),
+              new HostColumnVector.BasicType(true, DType.INT32)).asJava)
+        withResource(GpuColumnVector.from(ColumnVector.fromLists(
+          new HostColumnVector.ListType(true, structType), list1),
+          MapType(DataTypes.IntegerType, DataTypes.IntegerType))) { input =>
+          val batch = new ColumnarBatch(List(input).toArray, 1)
+          checkEvaluation(gpuToPrettyStr, expected0, batch)
+        }
+    }
+  }
+
+  test("test show() Struct") {
+    val dataType = StructType(Seq(StructField("a", DataTypes.IntegerType),
+      StructField("b", DataTypes.IntegerType),
+      StructField("c", DataTypes.IntegerType)))
+    val v = Array(1, 2, 3)
+    val cpuResult = executeOnCpuAndReturn(dataType, InternalRow(v: _*))
+    val child = GpuBoundReference(0, dataType, true)(NamedExpression.newExprId, "struct_arg")
+    val gpuToPrettyStr = GpuToPrettyString(child, Some("UTC"))
+    val l =
+      List[HostColumnVector.DataType](
+        new HostColumnVector.BasicType(false, DType.INT32),
+        new HostColumnVector.BasicType(false, DType.INT32),
+        new HostColumnVector.BasicType(false, DType.INT32),
```

Comment:
Trailing comma causes scalastyle to fail

```suggestion
        new HostColumnVector.BasicType(false, DType.INT32)
```